### PR TITLE
Build on windows

### PR DESCRIPTION
**Pull request by jtnord**
The unit test was broken on windows as the environment variables where not expanded

@reviewbybees

### CODE REVIEW

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

**Comment by recena** [comment]
:bee:

**Comment by tfennelly** [comment]
:bee:

**Comment by jtnord** [comment]
@reviewbybees done

**Comment by ghost** [comment]
This pull request has completed our internal processes and we now respectfully request the maintainers of this repository to consider our proposal contained within this pull request for merging.

File: src/test/java/hudson/plugins/batch_task/BatchTaskTest.java [hunk 1]
```diff
@@ -65,8 +69,14 @@ public void testExecute() throws Exception {
         hudson.getNodeProperties().add(new EnvironmentVariablesNodeProperty(
                 new EnvironmentVariablesNodeProperty.Entry("OVERRIDE_ME", "bar")));
         FreeStyleProject p = createFreeStyleProject("execute");
-        BatchTask task = new BatchTask("test",
-                "echo \"$TASK_ID:$GLOBAL:$OVERRIDE_ME:$HUDSON_USER\"\n");
+        BatchTask task;
+        if (Functions.isWindows()) {
+            task = new BatchTask("test",
+                    "echo \"%TASK_ID%:%GLOBAL%:%OVERRIDE_ME%:%HUDSON_USER%\"");        }
```
Comment (jtnord) [inline src/test/java/hudson/plugins/batch_task/BatchTaskTest.java]: what is this formatting evilness I see before me...

**Pull request merged**